app/views/users/index.html.erb

```diff
@@ -1,2 +1,12 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+
+<ul id='users_section'>
+  <% @users.each do |user| %>
+    <li class='users'>
+      <div class='users_pic'></div>
+      <div class='users_name'>
+          <h2>User <%= user.name %></h2>
+          <p>Number of posts: <%= user.posts_count %></p>
+      </div>
+    </li>
+  <% end %>
```

Comment:
**Changes Required**
- [ ] Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png). Let us try and make these changes